android/src/main/java/com/swmansion/rnscreens/SearchBarManager.kt

```diff
@@ -71,45 +84,33 @@ class SearchBarManager : ViewGroupManager<SearchBarView>() {
     }
 
     @ReactProp(name = "placeholder")
-    fun setPlaceholder(view: SearchBarView, placeholder: String?) {
+    override fun setPlaceholder(view: SearchBarView, placeholder: String?) {
         if (placeholder != null) {
             view.placeholder = placeholder
         }
     }
 
     @ReactProp(name = "textColor", customType = "Color")
-    fun setTextColor(view: SearchBarView, color: Int?) {
+    override fun setTextColor(view: SearchBarView, color: Int?) {
         view.textColor = color
     }
 
     @ReactProp(name = "headerIconColor", customType = "Color")
-    fun setHeaderIconColor(view: SearchBarView, color: Int?) {
+    override fun setHeaderIconColor(view: SearchBarView, color: Int?) {
         view.headerIconColor = color
     }
 
     @ReactProp(name = "hintTextColor", customType = "Color")
-    fun setHintTextColor(view: SearchBarView, color: Int?) {
+    override fun setHintTextColor(view: SearchBarView, color: Int?) {
         view.hintTextColor = color
     }
 
     @ReactProp(name = "shouldShowHintSearchIcon")
-    fun setShouldShowHintSearchIcon(view: SearchBarView, shouldShowHintSearchIcon: Boolean?) {
-        view.shouldShowHintSearchIcon = shouldShowHintSearchIcon ?: true
+    override fun setShouldShowHintSearchIcon(view: SearchBarView, value: Boolean) {
+        view.shouldShowHintSearchIcon = value ?: true
     }
```

Comment:
```suggestion
    override fun setShouldShowHintSearchIcon(view: SearchBarView, shouldShowHintSearchIcon: Boolean) {
        view.shouldShowHintSearchIcon = shouldShowHintSearchIcon ?: true
    }
```

If we can, we should stay with more expressive parameter names here.